.circleci/config.yml

```diff
@@ -72,9 +72,15 @@ jobs:
             - ~/workbench/ui/node_modules
           key: ui-cache-{{ checksum "~/workbench/ui/package.json" }}
       - run:
+          name: Run Angular tests
           working_directory: ~/workbench/ui
           command: |
-            npm run codegen && npm run build && npm test -- --no-watch
+            npm run codegen && npm test -- --no-watch
+      - run:
+          name: Test compilation of the Angular app
+          working_directory: ~/workbench/ui
+          command: |
```

Comment:
There's no need for pipes here, is there?  These commands are pretty comfortable on a single line.

Reply:
Yep, done.